### PR TITLE
Add Manjunath to people page

### PR DESCRIPTION
**Pull request by manjnrao**
Added a picture and added details on the people page.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -71,6 +71,12 @@
    email: ander.aranfdz@gmail.com
    pic: ander.jpg
 
+ - name: Manjunath Rao
+   role: grad
+   email: manjunath.rao@colorado.edu
+   linkedin: https://www.linkedin.com/in/manjunath-rao-n/
```
Comment (joewie) [inline _data/people.yml:77]: This shouldn't be a full URL, otherwise it breaks the link.

**Comment by joewie** [comment]
LinkedIn is saying the profile doesn't exist?

**Comment by joewie** [comment]
> LinkedIn is saying the profile doesn't exist?
> 
> 

I guess your privacy settings require users to be signed in to see your profile. That's fine, but it's probably worth considering making it public.

**Comment by manjnrao** [comment]
> > LinkedIn is saying the profile doesn't exist?
> 
> I guess your privacy settings require users to be signed in to see your profile. That's fine, but it's probably worth considering making it public.

Yeah, it's been private until now. I'll consider making it public.

**Comment by joewie** [comment]
> > > LinkedIn is saying the profile doesn't exist?
> > 
> > 
> > I guess your privacy settings require users to be signed in to see your profile. That's fine, but it's probably worth considering making it public.
> 
> Yeah, it's been private until now. I'll consider making it public.

After all, having that bar doesn't really give you much additional privacy—a LinkedIn account isn't exactly hard to acquire.

**Pull request merged**